### DISCUSSION TITLE
Denny rights

### DISCUSSION THREAD

**Discussion by ALIENQuake** [WeblateOrg/weblate#6083] [Q&A] [answered]
Hi,

I've able to do some experimentation with the permission system on my own but I at the point when I'm feeling that the things that I want simply aren't possible.

So, do weblate rights management system allows for:

- ensure that the current weblate instance is open for new people: for every project, component and language people can provide translations directly, add new languages, basically 'Power User' rights (default one)

- but at the same time, for a certain combination of component (or project) + specific language, everyone should be limited only to providing suggestions unless they are a member of a specific group, that can act as having 'Power User' rights

I can provide details about the context later why such 'deny' rights are needed. Right now the question is: can this be done?

**Comment by nijel** [comment, marked as answer]
Restriction on project level depends on project setting: https://docs.weblate.org/en/latest/admin/access.html#project-access-control

Restriction on component level is also possible: https://docs.weblate.org/en/latest/admin/projects.html#component-restricted
